reports/report_builder.py

```diff
@@ -1759,6 +1759,62 @@ def ws_68b(self, ws):
             self.tabulate(ws, counts, TOPICS, AGREE_DISAGREE, row_perc=True, write_col_headings=False, r=r)
             r += len(AGREE_DISAGREE)
 
+    def ws_72(self, ws):
+        """
+        Cols: Focus Topic, Media
+        Rows: Country
+        Focus: female reporters
+        """
+        # TODO: these values should be %age total of media in country/region
+
+        secondary_counts = OrderedDict()
+        for topic_id, topic in FOCUS_TOPICS.iteritems():
+            actual_topic_ids = [k for k, v in FOCUS_TOPIC_MAP.iteritems() if v == topic_id]
```

Comment:
I've reversed the map so that it's clearer what's going on. We weren't using this map effectively anyway.